Dockerfile.pgbouncer

```diff
@@ -0,0 +1,36 @@
+FROM debian:bullseye
```

Comment:
Is this bespoke, or is this dockerfile copied from somewhere?

Reply:
@chainlink comes from our pachyderm pgbouncer repo. just moving it to core. plan to release the new image, but do a round of testing before bumping it in our helm.

Reply:
Right, but we're not using that currently correct (currently still using bitnami?)

Reply:
yea a bitnami image that was pulled and pushed. we are just going to rebuild same version of it. but need to do enough testing to feel this "new image" is good to update the helm value ref